workflow-ui/compose/src/androidTest/java/com/squareup/workflow1/ui/compose/NoTransitionBackStackContainer.kt

```diff
@@ -21,28 +20,25 @@ import com.squareup.workflow1.ui.container.R
 internal class NoTransitionBackStackContainer(context: Context) : BackStackContainer(context) {
 
   override fun performTransition(
-    oldViewMaybe: View?,
-    newView: View,
+    oldHolderMaybe: ScreenViewHolder<NamedScreen<*>>?,
+    newHolder: ScreenViewHolder<NamedScreen<*>>,
     popped: Boolean
   ) {
-    oldViewMaybe?.let(::removeView)
-    addView(newView)
+    oldHolderMaybe?.view?.let(::removeView)
+    addView(newHolder.view)
   }
 
   companion object : ScreenViewFactory<BackStackScreen<*>>
-  by ManualScreenViewFactory(
-    type = BackStackScreen::class,
-    viewConstructor = { initialRendering, initialEnv, context, _ ->
-      NoTransitionBackStackContainer(context)
+  by ScreenViewFactory.forBuiltView(
+    buildView = { _, initialEnvironment, context, _ ->
+      val view = NoTransitionBackStackContainer(context)
         .apply {
           id = R.id.workflow_back_stack_container
           layoutParams = LayoutParams(MATCH_PARENT, MATCH_PARENT)
-          bindShowRendering(
-            initialRendering,
-            initialEnv,
-            { newRendering, newViewEnvironment -> update(newRendering, newViewEnvironment) }
-          )
         }
+      ScreenViewHolder(initialEnvironment, view) { rendering, environment ->
```

Comment:
Notice how `bindShowRendering` calls migrate to `ScreenViewHolder` instantiation.